app/views/pages/components/authors/author-entry.ejs

```diff
@@ -3,6 +3,7 @@
     <div>
       <%- include(`../../../ssrUtils/responsiveImage`, {
         url: `/image/${author.link}`,
+        version: author.lastChanged,
```

Comment:
Won't it invalidate cache even if author didn't change picture but something else on his profile

Reply:
lastChanged is attached to avatar change only